index.html

```diff
@@ -454,16 +457,33 @@ <h3>Character</h3>
 ...
         </pre>
 
-            <li>If the <a>Character</a> has a <a>Style</a>, the following TTML constraints apply:
+            <li>A <a>Character Style</a> is represented in TTML with the following structure and constraints:
               <ul>
-                <li>There MAY be a <code>style</code> element in the <code>styling</code> element providing the styles applicable to <a>Script Events</a> from this <a>Character</a>.
-                  The <code>xml:id</code> MUST be present.
-                  Any style attribute permitted in [[ttml-imsc1.2]] may be present.
-                  A <code>ttm:agent</code> attribute MAY be present to link the <code>ttm:agent</code> element representing the <a>Character</a>.
-                </li>
+                <li>Each <a>Character Style</a> is represented by one or more <code>&lt;style&gt;</code> element children of the <code>&lt;styling&gt;</code> element.</li>
+                <li>Each such <code>&lt;style&gt;</code> element is associated with the <a>Character</a> by having a <code>ttm:agent</code> attribute
+                  whose value is the <code>xml:id</code> of the <code>&lt;ttm:agent&gt;</code> element representing the <a>Character</a>.</li>
+                <li>A <a>Script Event</a> MAY apply <a>Character Styles</a> by including the <code>xml:id</code> of each style
+                  in the <code>style</code> attribute of the <code>&lt;div&gt;</code> element that defines that Script Event.</li>
+                <li>A <a>Text</a> object MAY apply <a>Character Styles</a> by including  the <code>xml:id</code> of each style
+                  in the <code>style</code> attribute of the <code>&lt;p&gt;</code> element that defines that Text object.</li>
+                <li>A <a>Script Event</a> SHOULD NOT apply a <a>Character Style</a> for a <a>Character</a> that is not associated with that Script Event.</li>
+                <li>A <a>Text</a> object SHOULD NOT apply a <a>Character Style</a> for a <a>Character</a> that is not associated with that Text object's Script Event.</li>
               </ul>
             </li>
 
+            <p class="note">Any style attribute defined in [[TTML2]] or [[ttml-imsc1.2]]
+              (or other profiles using non-W3C namespaces) can be present on the <code>&lt;style&gt;</code> element.</p>
+            <p>A <code>&lt;style&gt;</code> element MAY omit the <code>ttm:agent</code> attribute if it is not associated with a <a>Character</a>.
+              Such styles MAY be applied in the same way as any other style, via a reference in the <code>style</code> attribute.</p>
+            <p>Character Styles are applied to <a>Script Events</a> and <a>Text</a> by using the <code>style</code> attribute to specify the set of applicable styles.
+              Presentation Processors MUST NOT apply character styles to text if they are not specified using the <code>style</code> attribute.
```

Comment:
I would like to avoid using the term "Presentation Processor" as it leads to complex TTML2 terminology. This is related to https://github.com/w3c/dapt/issues/44.

Reply:
Should we resolve it when we resolve #44 then, and leave as is for now? I could add an Ed. note or an issue reference so we don't forget it.

Reply:
Fine resolving it in #44

Reply:
Added issue link as a reminder, in 185e0f6.